charts/nextcloud/templates/deployment.yaml

```diff
@@ -408,20 +408,30 @@ spec:
       {{- if .Values.nginx.enabled }}
       # Will mount configuration files as www-data (id: 82) for nextcloud
       securityContext:
-        fsGroup: 82
-        {{- if .Values.securityContext }}
+        {{- if .Values.nextcloud.securityContext }}
+        {{- with .Values.nextcloud.securityContext }}
```

Comment:
I would call that `nextcloud.podSecurityContext` and also add a container securityContext as `nextcloud.securityContext`.